### DISCUSSION TITLE
Reconsider allowing for an extra folder level

### DISCUSSION THREAD

**Discussion by sidney-eliot** [gotson/komga#1767] [Ideas]
In 2020 a similar request was closed https://github.com/gotson/komga/issues/185, the person there was asking for it for chapters in manga. I however think there is another reason why this should be added. Many of us structure our comic / manga / book libraries by artist / author name. This makes it easy to keep track of what one has from the folder structure and easily find things.

You also mentioned that adding and maintaining a new scanner is not hard, so please if you aren't willing to implement this feature, at the very least add a scanner type that uses the folder structure 1 to 1. Many of us have clean libraries where this will work nicely. Doing so will also fix many others folder structure problems, like the manga chapter person form the issue above.

@BryanWall also nicely puts it "... This is the main thing preventing me from using Komga. My content is already in a folder structure that makes everything easy to find and I would prefer the ability to view it (and access it through OPDS) using the folder structure. ..."

**Comment by gotson** [comment]
there's no plans for this, nor to reconsider this for now.
